### ISSUE TITLE
Request: CHiP Mesh

### ISSUE THREAD

**Issue by markstinson** [tomhiggins/anyfesto#12]
I just discovered this PirateBox post ...

https://piratebox.cc/raspberry_pi:mods:mesh

Given that CHiP has 2 built-in wlan (instead of having to use a USB dongle like a RPi), I would think it possible to make mesh an option for communication between nodes -- and easier considering it consistent chipset for all original Pocket/CHiPs (not CHiP Pro).

This should make it possible for local mesh with P2P between nodes.

I'm looking into the use case for larger off grid sites to have a several (solar/battery) powered CHiP network folks can connect their phones and tablets to for BBS, media library, WebChat (IRC), etc.

It will likely be a while before I make any progress due to end of year holidays.

**Comment by tomhiggins** [comment]
Great to hear. Yeah, the two seemingly workable radios on the chip is a nice thing. I played around with it some early on but I dropped it to get the core  functions stable. 
One caveat, the CHIP needs power. If you are using battery power you will most likely get  dropouts and reboots when using the two radios and also if you have a large usb mem stick on it. I spent aa fiar amount of time battling that till I had both the battery and the power cable plugged in, they work well to keeping the radios and usb up without blacking out. 
I look forward to seeing what you do with this. Do not worry about fast dev, this project has been on a slow but steady roll:)-
-tom
